test-suite/README.md

```diff
@@ -16,11 +16,10 @@
     9. Close Settings menu
 
 ## Setting up test-suite adaptors to run Postman tests
-1. Open project in IDE
-2. Use the terminal and CD into folder `\nia-patient-switching-standard-adaptor\test-suite`
-3. Run the code `sudo ./start-test-environment.sh`
-4. Allow the system to build adaptors
-5. After building your docker suite should look something like this (containers can appear in any order):
+1. Use the terminal and CD into folder `\nia-patient-switching-standard-adaptor\test-suite`
+2. Run the code `sudo ./start-test-environment.sh`
```

Comment:
You mention to run the code but "sudo" is Linux specific. Should we be specificying that in the instructions?

Reply:
I think `sudo` is only necessary when the file isn't flagged as executable on the system. I've moved it to the troubleshooting section. It should always work because its a bash script, so even on windows we run it using WSL and mac is now zsh which is an extension of bash